ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -306,7 +306,7 @@ export const Homepage = fp.flow(
             <FadeBox style={styles.fadeBox}>
               {/* The elements inside this fadeBox will be changed as part of ongoing homepage redesign work */}
               <FlexColumn style={{ justifyContent: 'flex-start' }}>
-                <Workspaces />
+                <Workspaces onChange={() => this.loadWorkspaces()} />
```

Comment:
How this works:
`loadWorkspaces()` updates the state of `userWorkspacesResponse` which is what `RecentResources` uses to populate its table